TestApp/TestApp/CustomViews/AtomicComponents/Molecules/VariableSizeLabelView.swift

```diff
@@ -0,0 +1,130 @@
+//
+//  VariableSizeLabelView.swift
+//  ECCBWallet
+//
+//  Created by Jonathan Bott on 11/1/19.
+//  Copyright © 2019 Jon Bott. All rights reserved.
+//
+
+import Foundation
+import DesignSystem
+
+public class VariableSizeLabelView: UIView {
+
+    private var smallNumbersDefaultedToZero = true
+
+    public enum Style: String {
+        
+        
+        case `default`,
+              numPadOutput,
+              numPadOutputSmall,
+              numPadOutputSmallDefault,
+              accentRequest
+
+        public var typography: Typography {
+            switch self {
+            case .numPadOutput:
+                return .x5large
+            case .numPadOutputSmall, .numPadOutputSmallDefault:
+                return .xxxlarge
+            case .accentRequest:
+                return .x4large
+            default: //default
```

Comment:
you can have finite switches now if you cover all cases.. no need for defaults anymore